components/led_panel.py

```diff
@@ -0,0 +1,95 @@
+import wpilib
+
+
+class LEDPanel:
+    # Key: Match number, Value: Drop Bears match number (<=15)
+    # TODO: Fill with proper values when the draw comes out
+    matches: dict[int, int] = {1: 1}
+
+    def __init__(self) -> None:
+        # Custom packet: 00 000000
+        # match state, state specific information
+        self.match_state = 0
+        self.packet = 0b01000000
+        self.panel_port = wpilib.SerialPort(
+            baudRate=9600, port=wpilib.SerialPort.Port.kUSB1, dataBits=8
+        )
+        self.set_match_id()
```

Comment:
This happens on robot code start. `getMatchNumber` will always be invalid in this scenario, as the Driver Station will not have connected to the robot.